controllers/etcd/config.go

```diff
@@ -15,9 +15,11 @@
 package etcd
 
 import (
-	"flag"
+	flag "github.com/spf13/pflag"
+	"k8s.io/component-base/featuregate"
 
 	"github.com/gardener/etcd-druid/controllers/utils"
+	"github.com/gardener/etcd-druid/pkg/features"
```

Comment:
Reorder imports - first group should contain standard libraries, next group should contain packages from within the project, next group should contain external packages.